src/app/modules/ix-tables/components/ix-table/ix-table.component.ts

```diff
@@ -39,6 +39,7 @@ import { MatTable } from '@angular/material/table';
   ],
   encapsulation: ViewEncapsulation.None,
   // See note on CdkTable for explanation on why this uses the default change detection strategy.
+  // eslint-disable-next-line @angular-eslint/prefer-on-push-component-change-detection
```

Comment:
Why do we need it?

Reply:
Because apparently according to the comment above Default change detection strategy is required for `CdkTable`. i don't know specifics, but decided not to override it.